src/programs/CMakeLists.txt

```diff
@@ -15,3 +15,6 @@ add_subdirectory(find_particles)
 add_subdirectory(local_resolution)
 add_subdirectory(convert_eer_to_mrc)
 add_subdirectory(azimuthal_average)
+if( BUILD_EXPERIMENTAL_FEATURES)
+    add_subdirectory(simulate)
+endif()
```

Comment:
github is flagging that there's no newline at the end of the file. Not sure if this has an effect.

Reply:
I had noticed this in a collaborators code - it looks like I need to add to settings.json

"html.format.endWithNewline": true